corehq/apps/locations/views.py

```diff
@@ -824,10 +824,7 @@ def post(self, request, *args, **kwargs):
             expiry=ONE_HOUR,
             file_extension=file_extention_from_filename(upload.name),
         )
-        task = import_locations_async.delay(
-            domain,
-            file_ref.download_id,
-        )
+        task = import_locations_async.apply_async(args=[domain, file_ref.download_id], countdown=10)
```

Comment:
Great catch! Is it worth adding the explanation for countdown here as a comment, or is it obvious?